CRM/Profile/Page/MultipleRecordFieldsListing.php

```diff
@@ -449,6 +455,13 @@ public function browse() {
         }
       }
     }
+
+    if (!CRM_Core_BAO_CustomGroup::isCustomGroupAllowed(
```

Comment:
This PR works well overall, but the logic on this `if` statement is inverted.

Reply:
Hi @MegaphoneJon 

Good spot. Have just pushed fix.

Thanks for reviewing and doing QA.
Pradeep